bin/helpers/utils.js

```diff
@@ -266,15 +266,53 @@ exports.setUserSpecs = (bsConfig, args) => {
   }
 }
 
-// env option must be set only from command line args as a string
 exports.setTestEnvs = (bsConfig, args) => {
+  let envKeys = {};
+
+  if(bsConfig.run_settings.env && Object.keys(bsConfig.run_settings.env).length !== 0) {
```

Comment:
Can you please update the template of browserstack.json?

Reply:
Synced. No need to add for optional capabilities. Will be added to docs.